blockstore/apps/api/v1/views/snapshots.py

```diff
@@ -1,18 +1,66 @@
 """
 Views for Snapshots.
+
+We need overall identifiers to make it clear what we're talking about and
+extract things out of it.
 """
+import codecs
+import uuid
 
 from django import http
-from rest_framework import status
-from rest_framework import viewsets
+from rest_framework import exceptions, status, viewsets
 from rest_framework.response import Response
 
-from blockstore.apps.bundles.store import BundleDataStore, BundleSnapshot
+from blockstore.apps.bundles.store import SnapshotRepo, Snapshot
 from blockstore.apps.bundles.models import BundleVersion
 
-from ...constants import FILE_PATH_REGEX
+from ...constants import FILE_PATH_REGEX, SNAPSHOT_HASH_DIGEST, UUID4_REGEX
 from ...utils import ZippableMultiValueDict
-from ..serializers.snapshots import FileInfoSerializer
+from ..serializers.snapshots import FileInfoSerializer, SnapshotSerializer
+
+
+class SnapshotViewSet(viewsets.ViewSet):
+    """
+    The Snapshots resource can only be accessed in a read-only detail view.
```

Comment:
Is there an example use case where, say, Studio might want to query a snapshot's details via this API? Or is this mostly for helping with blockstore implementation?

Reply:
But yeah, it's useful in the short term just for implementation. I'm not averse to stripping it out afterwards if we don't end up using it.

Reply:
You know what, you're right. I'm just going to remove this.